src/styles/card.scss

```diff
@@ -0,0 +1,102 @@
+.card {
+  width: $card-width;
```

Comment:
Ensure that the variable `$card-width` is defined in your SCSS files. If it's not defined, this will cause a compilation error.